generate/generate.go

```diff
@@ -782,6 +782,25 @@ func (s *service) generateHelperFuncs(a *API) {
 			pn("		return nil, -1, err")
 			pn("	}")
 			pn("")
+			pn("    if l.Count == 0 {")
+			pn("		// look	inside projects ")
+			pn("		p.p[\"projectid\"] = \"-1\"")
+			pn("		l2, err := s.ListVirtualMachines(p)")
```

Comment:
Ow, sorry... This should not contain the literal text `VirtualMachines` of course! So it should be like:

```go
pn("        l, err = s.List%s(p)", , ln)
```